tools/experiments/experiments.js

```diff
@@ -538,10 +538,29 @@ function updateExperimentRow(experiment) {
 function isExperimentOn_(id) {
   if (id == CANARY_EXPERIMENT_ID) {
     return getCookie(window, 'AMP_CANARY') == '1';
+  } else if (id == RC_EXPERIMENT_ID) {
+    return getCookie(window, 'AMP_RC') == '1';
```

Comment:
Can we not introduce another cookie?

Reply:
Is there a better way? What if I re-purpose (and rename?) the existing cookie into a string with values `canary` or `rc`?

Reply:
I would create an enum: `enum { 1, canary, rc }`. `1` and `canary` are both canary, and `rc` is rc.

>  One thing I can do is to ensure that only one of these two cookies can be set at any given time. WDYT?

If we got this route, definitely.

Reply:
What you suggest won't work because the code that looks for the cookie (as currently written) returns the Canary runtime if the `AMP_CANARY` cookie is non-empty.

Reply:
The more I think about it, given that all the experiments are on/off switches, my preference is to have one cookie per experiment. I've sent out a new commit that ensures that only one of `AMP_CANARY` and `AMP_RC` can be enabled at any given time.

Reply:
Which code uses non-empty AMP_CANARY?

Reply:
It's in internal code. Will follow up offline.

Reply:
Discussed offline, I've sent out a new commit that rolls both opt-in states into `AMP_CANARY`. `0` is disabled, `1` is canary, `2` is rc.